bench/wtperf/runners/wtperf_run.sh

```diff
@@ -99,6 +99,11 @@ while test "$run" -le "$runmax"; do
 	if test "$?" -ne "0"; then
 		exit 1
 	fi
+
+	# Copy the artifacts from the run
+	backup_dir=${home}_$(basename $wttest)_${run}_$(date +"%s")
+	rsync -r -m --exclude=*Log* --exclude=*.wt --exclude=WiredTiger --exclude=*Preplog* --exclude=WiredTiger.lock --exclude=WiredTiger.turtle $home/ $backup_dir
+
```

Comment:
I think if we wanted any archiving across runs, that logic should be in Jenkins (like what we have for test failure tarballs).